### PR TITLE
Cell ewas methods

### PR DESCRIPTION
**Pull request by ejh243**
# Description

This pull request will update scripts in line with methods paper draft manuscript

## Type of pull request

- [ ] Bug fix
- [X] New feature/enhancement
- [ ] Code refactor
- [ ] Documentation update

## Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have tested my code to check that it is functional
- [ ] I have used linters to check for common sources of errors
- [ ] I have implemented fail safes in my code to account for edge cases
- [ ] I have made the corresponding changes to the documentation

### CODE REVIEW

File: array/DNAm/analysis/methodsDevelopment/simulateCellSpecificEWAS.r [hunk 1]
```diff
@@ -231,62 +234,64 @@ for(simNum in 1:nSim){
 			outtab.sim<-outtab.null
 			outtab.sim[sigProbes,]<-outtab.sig
 			
-			sumSim[rowNum,2+seq(1,6*9,6)]<-colSums(outtab.sim < 9e-8)
-			sumSim[rowNum,3+seq(1,6*9,6)]<-colSums(outtab.sim[sigProbes,] < 9e-8)
-			sumSim[rowNum,4+seq(1,6*9,6)]<-colSums(outtab.sim[-sigProbes,] < 9e-8)
-			
-			# substract those significant in both ME and Int from ME count
```
Comment (ew367) [inline array/DNAm/analysis/methodsDevelopment/simulateCellSpecificEWAS.r:238]: What do ME and Int stand for?

**Review by ew367 — APPROVED** [review]
It would be nice to have a read.me doc that gives an overview of the pipeline / order of scripts for the analysis, but I can't see anything that should prevent these changes being merged.

**Comment by ejh243** [comment]
> It would be nice to have a read.me doc that gives an overview of the pipeline / order of scripts for the analysis, but I can't see anything that should prevent these changes being merged.

Yes, my intention is to add that.

File: array/DNAm/analysis/methodsDevelopment/simulateCellSpecificEWAS.r (hunk 1 above)
Comment (ejh243) [reply array/DNAm/analysis/methodsDevelopment/simulateCellSpecificEWAS.r:238]: ME stands for main effect and Int interaction. They refer to the two terms in the linear model used to identify DMPs.

**Comment by sof202** [comment]
> > It would be nice to have a read.me doc that gives an overview of the pipeline / order of scripts for the analysis, but I can't see anything that should prevent these changes being merged.
> 
> Yes, my intention is to add that. 

That can be in another pull request though.

**Pull request merged**